crypto/argon2/blake2b.c

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright 2016-2019 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include "blake2b.h"
+#include <prov/blake2.h>
+#include "openssl/core_names.h"
+#include "openssl/params.h"
+
+int blake2b(void *out, size_t outlen, const void *in, size_t inlen,
+            const void *key, size_t keylen)
+{
+    int ret = 0;
+
+    EVP_MAC *mac;
```

Comment:
initialize these variables to NULL then you can have just a single "goto" label.